2018-07-20 15:08:34,312 ERROR: Exception on /index [GET] [in /usr/lib/python3/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/nate/.local/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/nate/BadDoggy/Artemis/app/routes.py", line 28, in index
    return render_template('index.html', title='Home Page',form=form, posts=posts)
  File "/usr/lib/python3/dist-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/usr/lib/python3/dist-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/nate/BadDoggy/Artemis/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/nate/BadDoggy/Artemis/app/templates/base.html", line 22, in top-level template code
    {% block content %}{% endblock %}
  File "/home/nate/BadDoggy/Artemis/app/templates/index.html", line 17, in block "content"
    {% for post in posts %}
TypeError: 'Pagination' object is not iterable
2018-07-20 15:08:41,300 ERROR: Exception on /index [GET] [in /usr/lib/python3/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/nate/.local/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/nate/BadDoggy/Artemis/app/routes.py", line 28, in index
    return render_template('index.html', title='Home Page',form=form, posts=posts)
  File "/usr/lib/python3/dist-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/usr/lib/python3/dist-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/nate/BadDoggy/Artemis/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/nate/BadDoggy/Artemis/app/templates/base.html", line 22, in top-level template code
    {% block content %}{% endblock %}
  File "/home/nate/BadDoggy/Artemis/app/templates/index.html", line 17, in block "content"
    {% for post in posts %}
TypeError: 'Pagination' object is not iterable
2018-07-20 15:27:17,906 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:42]
2018-07-20 15:27:43,050 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:42]
2018-07-21 15:29:58,826 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-21 15:34:29,756 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-21 15:37:23,217 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-25 19:03:16,558 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-25 19:04:27,329 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-25 19:05:38,694 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-25 19:06:35,192 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-25 19:08:46,430 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-25 19:11:18,631 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:44]
2018-07-26 15:28:46,308 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:45]
2018-07-26 15:31:19,373 ERROR: Exception on /index [GET] [in /usr/lib/python3/dist-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/nate/.local/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/nate/BadDoggy/Artemis/app/routes.py", line 31, in index
    prev_url=prev_url)
  File "/usr/lib/python3/dist-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/lib/python3/dist-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/nate/BadDoggy/Artemis/app/templates/index.html", line 15, in template
    <li class="previous{% if not prev_url %} disabled{% endif}">
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/usr/lib/python3/dist-packages/jinja2/parser.py", line 216, in parse_if
    token = next(self.stream)
  File "/usr/lib/python3/dist-packages/jinja2/lexer.py", line 359, in __next__
    self.current = next(self._iter)
  File "/usr/lib/python3/dist-packages/jinja2/lexer.py", line 562, in wrap
    for lineno, token, value in stream:
  File "/usr/lib/python3/dist-packages/jinja2/lexer.py", line 683, in tokeniter
    filename)
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
2018-07-26 15:33:16,998 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:45]
2018-07-26 16:19:32,542 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:46]
2018-07-26 16:35:01,688 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:46]
2018-07-26 16:40:53,068 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:46]
2018-07-26 16:41:18,497 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:46]
2018-07-28 18:07:41,789 INFO: Artemis Startup [in /home/nate/BadDoggy/Artemis/app/__init__.py:49]
